File: apps/plant_shop_java/plant_shop_java_active_web/src/models/Order.java
package models;

import org.javalite.activejdbc.Model;
import org.javalite.activejdbc.annotations.BelongsTo;
import org.javalite.activejdbc.annotations.HasMany;
import org.javalite.activejdbc.annotations.Table;

@Table("orders")
@BelongsTo(parent = User.class, foreignKeyName = "user_id")
@HasMany(child = OrderItem.class, foreignKeyName = "order_id")
public class Order extends Model {}

File: apps/plant_shop_c-sharp/plant_shop_asp_EF_core/Models/Order.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace plant_shop_asp_EF_core.Models
{
    [Table("orders")]
    public class Order
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Required]
        [Column("user_id")]
        public int UserId { get; set; }

        [Column("total", TypeName = "decimal(18, 2)")]
        public decimal TotalPrice { get; set; }

        [Column("status")]
        public string Status { get; set; } = "pending";

        [Column("created_at")]
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

        // Relations
        [ForeignKey("UserId")]
        public User? User { get; set; }
        public ICollection<OrderItem> OrderItems { get; set; } = new List<OrderItem>();
    }
}
